README.md

```diff
@@ -1,3 +1,69 @@
-# Skynet SDK
+# Skynet NodeJS SDK
 
-An SDK for integrating Skynet into Node.js applications
+An SDK for integrating Skynet into Node.js applications.
+
+## Installing
+
+Using `npm`:
+
+```sh
+npm install @nebulous/skynet
+```
+
+Using `yarn`:
+
+```sh
+yarn add @nebulous/skynet
+```
+
+## Examples
+
+### Uploading and downloading
+
+```js
+const skynet = require("@nebulous/skynet");
+
+const fs = require("fs");
+
+(async () => {
+  // upload file
+
+  const src_file = "./testdata/file1.txt";
+
+  const text = fs.readFileSync(src_file);
+  console.log(`Uploading file ${src_file} containing data: '${text}'`);
+
+  const skylink = await skynet.uploadFile(src_file, skynet.DefaultUploadOptions);
+  if (!skylink.includes(skynet.UriSkynetPrefix)) {
+    console.log(`ERROR: invalid skylink returned: ${skylink}`);
+    return;
+  }
+  console.log(`Upload successful, skylink: ${skylink}`);
+
+  // download file
```

Comment:
You could split download, upload and directory upload into 3 code blocks, each with it's own header, for better readability

Reply:
Removing the readme.